index.js

```diff
@@ -15,11 +17,65 @@ app.post("/", function(req, res, next) {
       var url = "https://calendar.google.com/calendar/embed?mode=AGENDA&amp;height=600&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=" + user + "%40goodeggs.com&amp;color=%2329527A&amp;ctz=America%2FLos_Angeles"
       res.status(200).send("Click to open <" + url + "|@" + user + "'s calendar>");
       break;
+
+    case "/helpscout":
+      var newConvo = req.body.text.split(" ");
+      var customer = newConvo[0];
+      var mailbox = newConvo[1];
+      var removeItems = newConvo.splice(0, 2);
+      var subject = newConvo.toString(",").replace(/,/g, " ");
+
+      if (mailbox === "it") {
+        mailbox = 19176;
+      } else {
+        mailbox = 98082;
```

Comment:
more mailboxes please! Also probably better to throw an error if the mailbox is unrecognized rather then just dumping them all in devops?